google/cloud/forseti/services/server_config/service.py

```diff
@@ -26,6 +29,48 @@
 LOGGER = logger.get_logger(__name__)
 
 
+def _run(client):
+    """Runs Forseti.
+
+    Args:
+        client (ClientComposition): Forseti client object.
+
+    Returns:
+        ServerRunReply: the returned proto message.
+    """
+    today = datetime.now()
+    model_name = today.strftime('run_model_%Y%m%d_%H%M')
+    message = 'Forseti server run complete'
+
+    inventory_id = 0
+
+    for progress in client.inventory.create():
+        inventory_id = progress.id
+
+    model_reply = client.model.new_model(
+        'inventory',
+        model_name,
+        inventory_id,
+        False)
+
+    model_handle = model_reply.model.handle
+    model_status = model_reply.model.status
+
+    client.switch_model(model_handle)
+
+    if model_status in ['SUCCESS', 'PARTIAL_SUCCESS']:
```

Comment:
The code would be easier to read if you go like this:
```
if model_status not in ['SUCCESS', 'PARTIAL_SUCCESS']:
    message = 'ERROR: Model status is {}'.format(model_status)
    return server_pb2.ServerRunReply(message=message)
```

Reply:
Yes